common/images/ome.svg

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="logo_-_color" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
+	 y="0px" viewBox="0 0 3568 1252" style="enable-background:new 0 0 3568 1252;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#DF283F;}
+	.st1{fill:#1C4A87;}
+	.st2{fill:#128669;}
+	.st3{fill:#1D8DCD;}
+</style>
+<g transform="scale(0.9) translate(178.4,178.4)">
+	<g>
+		<path class="st0" d="M256,448c0,70.7-57.3,128-128,128S0,518.7,0,448s57.3-128,128-128S256,377.3,256,448z"/>
+		<path class="st1" d="M1024,448c0,70.7-57.3,128-128,128s-128-57.3-128-128s57.3-128,128-128S1024,377.3,1024,448z"/>
+		<path class="st2" d="M832,128c0,70.7-57.3,128-128,128s-128-57.3-128-128S633.3,0,704,0S832,57.3,832,128z"/>
+		<path class="st1" d="M448,128c0,70.7-57.3,128-128,128s-128-57.3-128-128S249.3,0,320,0S448,57.3,448,128z"/>
+		<path class="st3" d="M832,768c0,70.7-57.3,128-128,128s-128-57.3-128-128s57.3-128,128-128S832,697.3,832,768z"/>
+		<path class="st1" d="M448,768c0,70.7-57.3,128-128,128s-128-57.3-128-128s57.3-128,128-128S448,697.3,448,768z"/>
+	</g>
+	<g>
+		<path class="st0" d="M1880.8,202.8c-67.1-65.6-150.1-98.3-248.9-98.3c-98.8,0-181.8,32.8-248.9,98.3
+			c-67.1,65.5-100.7,147.2-100.7,245.1S1315.9,627.5,1383,693c67.1,65.6,150.1,98.3,248.9,98.3s181.8-32.8,248.9-98.3
+			c67.1-65.6,100.7-147.2,100.7-245.1S1947.9,268.4,1880.8,202.8z M1866.6,447.9c0,66.8-22.7,123.7-67.9,170.5
+			c-45.3,46.9-100.9,70.3-166.7,70.3c-65.9,0-121.5-23.4-166.7-70.3c-45.3-46.9-67.9-103.7-67.9-170.5c0-66.8,22.6-123.6,67.9-170.5
+			c45.3-46.9,100.8-70.3,166.7-70.3c65.9,0,121.4,23.4,166.7,70.3C1843.9,324.3,1866.6,381.1,1866.6,447.9z"/>
+		<polygon class="st2" points="2707.3,127.3 2506.8,544.3 2306.4,127.3 2132,127.3 2132,791.3 2244,791.3 2244,272.6 2474.5,733.3 
+			2537.2,733.3 2769,272.6 2769,791.3 2881,791.3 2881,127.3 		"/>
+		<polygon class="st3" points="3202,685.3 3202,508.3 3520,508.3 3520,408.3 3202,408.3 3202,232.3 3557,232.3 3557,127.3 
+			3090,127.3 3090,791.3 3568,791.3 3568,685.3 		"/>
```

Comment:
No particular reason, we all copied and pasted the original SVG file which included this whitespace but this can probably be trimmed either here or in a follow-up PR.